problemsets-mat223/modules/module8.tex

```diff
@@ -23,7 +23,30 @@
 	where $\vec b_1=\xhat+\yhat$, and $\vec b_2 =3\yhat$ be another basis for $\R^2$. Given that $\vec v=2\xhat-\yhat$, 
 	find $[\vec v]_{\mathcal E}$ and $[\vec v]_{\mathcal B}$.
 
-	XXX Finish
+	Given that $\vec v=2\xhat-\yhat$, we see
+	\[
+	    [\vec v]_{\mathcal E}=\mat{2\\-1}.
+	\]
+	To find $[\vec v]_{\mathcal B}$, we can use a system of equations to find $\vec v$ as a linear combination of $\vec b_1$ and $\vec b_2$. That is, set 
+	\[
+	    \vec v = x\vec b_1 + y\vec b_2
+	\]
+	and substitute for $\vec v$, $\vec b_1$, and $\vec b_2$. This is equivalent to finding scalars $x$ and $y$ so that
+	\[
+	    \vec v = 2\xhat-\yhat=x(\xhat+\yhat)+3y\yhat=x\xhat+(x+3y)\yhat.
+	\]
```

Comment:
This is getting very close to finished, but it doesn't flow right. Read the whole thing as it is and make the sentences fit together a bit better.

Reply:
Ohhh I actually try to do that. But eventually I delete all of them and do as you said. Yeah I will do this